Marlin/Configuration_tenlog.h

```diff
@@ -2,6 +2,10 @@
 #define STR_HELPER(x) #x
 #define STR(x) STR_HELPER(x)
 
+#if __has_include("Configuration_overide.h") && __has_include(<stdint.h>)
```

Comment:
Was this needed to add here?  It is meant to be called last so that it has the final say in all variables.

Reply:
as for me it's good thing as I can than define something like
```#define SPIDER11
#define MachineTLD3P
#define MKS12864
#define Driver2209BTT
#define OpticalY
#define AllMetalHotend
#define ACBed
#define BMGExtruderV2
```
in `Configuration_overide.h` and it works
and don't need modify any other files at all to define exact printer

but I can remove this include, not a problem :)

Reply:
Yeah lets remove this from this PR, but if you'd like to create an additional file named `Machine_override.h' and submit that in a PR, it would be a handy feature.

Reply:
***edits

Reply:
Good idea... made optional Machine_overide.h include at the beginning